src/edtr-io/plugins/highlight/editor.tsx

```diff
@@ -37,50 +14,29 @@ export function HighlightEditor(props: HighlightProps) {
     if (!edit) {
       setTimeout(() => {
         setEditThrottled(false)
-      }, 500)
+      }, 100)
```

Comment:
@Entkenntnis do you know why this delay was implemented?
Will shortening it break something? (Feels less buggy with 100ms)

Reply:
mellie added it 4 years ago: https://github.com/edtr-io/edtr-io/commit/ec459ce6b843e61eb65ba56337284e126b89f94c

The plugin changed significantly since, currently I don't see anything that could break (react should take care of rendering) - maybe we don't need anymore

Reply:
@metzgaria do you by change rememer why this was needed? 😅